DataStore2/SavingMethods/OrderedBackups.lua

```diff
@@ -46,11 +46,11 @@ function OrderedBackups:Set(value)
 	local key = (self.mostRecentKey or 0) + 1
 
 	return Promise.async(function(resolve)
-		self.dataStore:SetAsync(key, value)

```

Comment:
`SetAsync` provides a third parameter to define user IDs which correlate to a specific piece of data.